docs/about/parameters.rst

```diff
@@ -0,0 +1,279 @@
+SmartPeak Parameters
```

Comment:
It's sad that this cannot be generated easily :sweat_smile:  we have the information spread in the code, but no easy solution.